app/views/prison/visits/_issue_with_prisoner_section.html.erb

```diff
@@ -64,13 +64,13 @@
     <div class="column-one-third">
       <%= error_container(rf, :reasons) do %>
         <label class="block-label">
-          <%= rf.object.checkbox_for(:prisoner_released, class: 'js-Conditional') %>
+          <%= rf.object.checkbox_for(:prisoner_released, class: 'js-Rejection', 'data-rejection-el': 'rejection-message') %>
           <%= t('.prisoner_released') %>
         </label>
       <% end %>
       <%= error_container(rf, :reasons) do %>
         <label class="block-label">
-          <%= rf.object.checkbox_for(:prisoner_banned, class: 'js-Conditional') %>
+          <%= rf.object.checkbox_for(:prisoner_banned, class: 'js-Rejection', 'data-rejection-el': 'rejection-message') %>
           <%= t('.prisoner_banned') %>
         </label>
       <% end %>
```

Comment:
There seems to be a repeating pattern here with only the field names changing. Could we have a loop go through an array of field names or use a partial here?